pom.xml

```diff
@@ -296,14 +474,6 @@
                         <source>1.8</source>
                         <target>1.8</target>
                         <release>8</release>
-                        <failOnWarning>true</failOnWarning>
-                        <showWarnings>true</showWarnings>
-                        <compilerArgs>
-                            <arg>-Xlint:all</arg>
-                            <!-- This fixes a boostrapclass path warning which the "release" property fixes on Java 9+ -->
-                            <arg>-Xlint:-options</arg>
-                            <arg>-Werror</arg>
-                        </compilerArgs>
                     </configuration>
```

Comment:
This should not be disabled. The code should be updated to not have warnings, or if there are specific issues that are impossible to work around they should be suppressed individually.

As it stands the current code won't run on a Java 8 VM due to the fact some Java 9 APIS are being used. (e.g. "List.of", "Map.of")